pkg/query-service/rules/resultTypes.go

```diff
@@ -15,7 +15,8 @@ type Vector []Sample
 type Sample struct {
 	Point
 
-	Metric labels.Labels
+	Metric    labels.Labels
+	IsMissing bool
```

Comment:
Where is `IsMissing` referenced apart from being populated? Maybe not a good idea to change `Sample` struct to handle `absent` case?

Reply:
I remembered why I added this. When the sample is missing the description and summary should be updated.